gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -384,14 +456,65 @@ def main(args):
         action="store_true",
     )
     parser.add_argument(
-        "--preprocess-data",
+        "--calculate-gerp-cutoffs",
         help=(
-            "Whether to prepare the exome, genome, and context Table for constraint"
-            " calculations by adding necessary coverage, methylation level, and VEP"
-            " annotations."
+            "Calculate GERP lower and upper cutoffs based on 5th and 95th percentiles"
+            " of GERP scores in the context Table. Note that if"
+            " '--calculate-gerp-cutoffs' is specified, the default values in"
+            " --gerp-lower-cutoff and --gerp-upper-cutoff will be overwritten."
```

Comment:
This will need reformatting

```suggestion
            " '--calculate-gerp-cutoffs' is specified, the computed values will be used in all downstream steps of the constraint pipeline instead of the values defined by --gerp-lower-cutoff and --gerp-upper-cutoff."
```